docs/output-data.md

```diff
@@ -2,24 +2,25 @@
 
 ## Displaying output data
 
-Each config item has an input_number or switch associated with it, you can find an auto generated dashboard for your
-configuration in your AppDaeamon configuration area under the filename **predbat_dashboard.yaml**
+Each predbat config item has an input_number or switch associated with it, you can find an auto-generated dashboard for your
+configuration in your AppDaemon configuration area under the filename **predbat_dashboard.yaml**.
+You will need to open that file in a Home Assistant file editor and copy the contents into a new dashboard page.
 
-You can also create a card using 'dynamic-entities-card.yaml' for a dynamically created list of entities for predbat which
+You can also create a dashboard page using [dynamic-entities-card.yaml](dynamic-entities-card.yaml) for a dynamically created list of all predbat entities which
 groups the entities by type and is collapsed by default to prevent screen clutter. Requires lovelace-collapsable-cards
 ([https://github.com/RossMcMillan92/lovelace-collapsable-cards](https://github.com/RossMcMillan92/lovelace-collapsable-cards))
 and lovelace-auto-entities ([https://github.com/thomasloven/lovelace-auto-entities](https://github.com/thomasloven/lovelace-auto-entities))
 to be installed via HACS as well as the stock vertical stack card. Credit @DJBenson for the code.
 
 ## Basic status
 
-- predbat.status - Gives the current status & errors and logs any adjustments made to your inverter
+- predbat.status - Gives the current status & errors and logs any changes that predbat makes to your inverter
```

Comment:
I'd make 'Predbat' upper case here, and there are quite a few other places in the PR too where it's in text (i.e. not a parameter name) and is lowercase.

Reply:
Good point, I've been through the docs again and corrected the capitalisations